mmocr/core/evaluation/utils.py

```diff
@@ -136,24 +136,32 @@ def points2polygon(points):
     return plg(point_mat)
 
 
-def poly_intersection(poly_det, poly_gt, invalid_ret=0, return_poly=False):
+def poly_intersection(poly_det, poly_gt, invalid='allow', return_poly=False):
```

Comment:
Invalid has multi meanings. Can we use two parameters? We also need check the invalid inputting legality.